pkg/operation/botanist/component/istio/monitoring.go

```diff
@@ -170,15 +170,15 @@ metric_relabel_configs:
 metrics_path: /stats/prometheus
 kubernetes_sd_configs:
 - role: endpoints
-  namespaces:
-    names: [ ` + v1beta1constants.DefaultSNIIngressNamespace + ` ]
+  selectors:
+   label: istio=ingressgateway
 relabel_configs:
 - source_labels:
   - __meta_kubernetes_service_name
   - __meta_kubernetes_endpoint_port_name
   - __meta_kubernetes_namespace
   action: keep
-  regex: ` + v1beta1constants.DefaultSNIIngressServiceName + `;tls-tunnel;` + v1beta1constants.DefaultSNIIngressNamespace + `
+  regex: ` + v1beta1constants.DefaultSNIIngressServiceName + `;tls-tunnel;` + v1beta1constants.DefaultSNIIngressNamespace + `.*
```

Comment:
What if https://github.com/gardener/gardener/blob/b2d32fe231cb47926480eb30a1d8f2ca6b9be4dc/example/20-componentconfig-gardenlet.yaml#L153 is changed?
What if https://github.com/gardener/gardener/blob/b2d32fe231cb47926480eb30a1d8f2ca6b9be4dc/example/20-componentconfig-gardenlet.yaml#L168 is changed?